content/cloud-docs/api-docs/configuration-versions.mdx

```diff
@@ -346,6 +352,38 @@ curl \
   https://archivist.terraform.io/v1/object/4c44d964-eba7-4dd5-ad29-1ece7b99e8da
 ```
 
+## Archive a Configuration Version
+
+`POST /configuration-versions/:configuration_version_id/actions/archive`
```

Comment:
Is this something we want to add a version note on for when this is actually available for enterprise side folks? e.g., like is done [here](https://www.terraform.io/cloud-docs/api-docs/plans#retrieve-the-json-execution-plan) for example